awx/main/access.py

```diff
@@ -1111,6 +1111,19 @@ def can_delete(self, obj):
         return self.can_change(obj, None)
 
 
+class CredentialInputSourceAccess(BaseAccess):
+    '''
+    I can see credential input sources when:
+     - I'm a superuser (TODO: Update)
+    I can create credential input sources when:
+     - I'm a superuser (TODO: Update)
+    I can delete credential input sources when:
+     - I'm a superuser (TODO: Update)
+    '''
+
+    model = CredentialInputSource
```

Comment:
There may not be much work here; most of the `BaseAccess` implementation actually _defaults_ to superuser-only.

Reply:
@AlanCoding do we actually need to _do_ anything here?  Don't these all just default to `superuser` only?